RDKit 2024.03.6 fails to build on Mac with certain options

**Describe the bug**

RDKit 2024.03.6 fails to build on Mac with certain options.

```sh
rdkit/Code/RDGeneral/RDExportMacros.h:19:10: fatal error: 'boost/config.hpp' file not found
#include <boost/config.hpp>
```

It doesn't fail when any options that call `find_package(Boost, ...)` are enabled, like `-DRDK_BUILD_PYTHON_WRAPPERS=ON`, `-DRDK_USE_BOOST_SERIALIZATION=ON`, etc.

**To Reproduce**

```sh
git clone --recursive --branch Release_2024_03_6 https://github.com/rdkit/rdkit.git
cd rdkit
cmake -S . -B build \
  -DRDK_BUILD_COORDGEN_SUPPORT=OFF \
  -DRDK_BUILD_MAEPARSER_SUPPORT=OFF \
  -DRDK_BUILD_PYTHON_WRAPPERS=OFF \
  -DRDK_USE_BOOST_IOSTREAMS=OFF \
  -DRDK_USE_BOOST_SERIALIZATION=OFF \
  -DRDK_USE_BOOST_STACKTRACE=OFF
cmake --build build
```

**Expected behavior**

The build succeeds.

**Configuration (please complete the following information):**
 - RDKit version: 2024.03.6
 - OS: macOS 14
 - Python version (if relevant): N/A
 - Are you using conda? No
 - If you are using conda, which channel did you install the rdkit from? N/A
 - If you are not using conda: how did you install the RDKit? Source

Thanks @greglandrum, here's a repro on GitHub Actions: https://github.com/ankane/rdkit-repro/actions/runs/10627243854/job/29460076854

```
-- The C compiler identification is AppleClang 15.0.0.15000309
-- The CXX compiler identification is AppleClang 15.0.0.15000309
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode_15.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode_15.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
RDK_OPTIMIZE_POPCNT is not available on aarch64 or arm64
-- Performing Test HAVE_FLAG__ffile_prefix_map__Users_runner_work_rdkit_repro_rdkit_repro_rdkit_build__deps_catch2_src__
-- Performing Test HAVE_FLAG__ffile_prefix_map__Users_runner_work_rdkit_repro_rdkit_repro_rdkit_build__deps_catch2_src__ - Success
-- Could NOT find Eigen3 (missing: EIGEN3_INCLUDE_DIR EIGEN3_VERSION_OK) (Required is at least version "2.91.0")
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
== Using strict rotor definition
Downloading https://github.com/rareylab/RingDecomposerLib/archive/v1.1.3_rdkit.tar.gz...
== Making EnumerateLibrary without boost Serialization support
== Making TautomerQuery without boost Serialization support
== Making FilterCatalog without boost Serialization support
-- Found Python3: /opt/homebrew/Frameworks/Python.framework/Versions/3.12/bin/python3.12 (found version "3.12.5") found components: Interpreter
Downloading https://github.com/google/fonts/raw/main/ofl/comicneue/ComicNeue-Regular.ttf...
Downloading https://github.com/google/fonts/raw/main/ofl/comicneue/OFL.txt...
-- Found Freetype: /opt/homebrew/lib/libfreetype.dylib (found version "2.13.3")
== Making SubstructLibrary without boost Serialization support
Downloading https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz...
-- Configuring done (11.0s)
-- Generating done (1.0s)
-- Build files have been written to: /Users/runner/work/rdkit-repro/rdkit-repro/rdkit/build
[  0%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/benchmark/catch_chronometer.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/benchmark/detail/catch_benchmark_function.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/benchmark/detail/catch_run_for_at_least.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/benchmark/detail/catch_stats.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/generators/catch_generator_exception.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/generators/catch_generators.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/generators/catch_generators_random.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_automake.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_common_base.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_compact.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_console.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_cumulative_base.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_event_listener.cpp.o
[  1%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_helpers.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_junit.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_multi.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_registrars.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_sonarqube.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_streaming_base.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_tap.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_teamcity.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/reporters/catch_reporter_xml.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_capture.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_config.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_exception.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_generatortracker.cpp.o
[  2%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_registry_hub.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_reporter.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_reporter_factory.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/interfaces/catch_interfaces_testcase.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_approx.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_assertion_result.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_config.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_get_random_seed.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_message.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_registry_hub.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_session.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_tag_alias_autoregistrar.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_test_case_info.cpp.o
[  3%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_test_spec.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_timer.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_tostring.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_totals.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_translate_exception.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/catch_version.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_assertion_handler.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_case_insensitive_comparisons.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_clara.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_commandline.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_console_colour.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_context.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_debug_console.cpp.o
[  4%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_debugger.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_decomposer.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_enforce.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_enum_values_registry.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_errno_guard.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_exception_translator_registry.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_fatal_condition_handler.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_floating_point_helpers.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_getenv.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_istream.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_lazy_expr.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_leak_detector.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_list.cpp.o
[  5%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_message_info.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_output_redirect.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_parse_numbers.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_polyfills.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_random_number_generator.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_random_seed_generation.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_reporter_registry.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_reporter_spec_parser.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_result_type.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_reusable_string_stream.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_run_context.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_section.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_singletons.cpp.o
[  6%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_source_line_info.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_startup_exception_registry.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_stdstreams.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_string_manip.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_stringref.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_tag_alias_registry.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_test_case_info_hasher.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_test_case_registry_impl.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_test_case_tracker.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_test_failure_exception.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_test_registry.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_test_spec_parser.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_textflow.cpp.o
[  7%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_uncaught_exceptions.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_wildcard_pattern.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/internal/catch_xmlwriter.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_container_properties.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_exception.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_floating_point.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_predicate.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_quantifiers.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_string.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/catch_matchers_templated.cpp.o
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2.dir/catch2/matchers/internal/catch_matchers_impl.cpp.o
[  8%] Linking CXX static library libCatch2.a
[  8%] Built target Catch2
[  8%] Building CXX object _deps/catch2-build/src/CMakeFiles/Catch2WithMain.dir/catch2/internal/catch_main.cpp.o
[  9%] Linking CXX static library libCatch2Main.a
[  9%] Built target Catch2WithMain
[  9%] Creating directories for 'Eigen'
[  9%] Performing download step (git clone) for 'Eigen'
Cloning into 'eigen'...
HEAD is now at 0fd6b4f Bump to 3.3.9
[  9%] Performing update step for 'Eigen'
-- Already at requested tag: 3.3.9
[  9%] No patch step for 'Eigen'
[  9%] No configure step for 'Eigen'
[ 10%] No build step for 'Eigen'
[ 10%] No install step for 'Eigen'
[ 10%] Completed 'Eigen'
[ 10%] Built target Eigen
[ 10%] Built target inchi_support
[ 10%] Built target coordgen_support
[ 10%] Built target ringdecomposerlib_support
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLapsp.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLbitset.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLcycleFams.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLdimacs.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLgraph.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLhandler.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLinfo.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLrelation.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLstack.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLtarjan.c.o
[ 10%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLtesting.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RDLutility.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib.dir/RingDecomposerLib/src/RingDecomposerLib/RingDecomposerLib.c.o
[ 11%] Linking C shared library ../../lib/libRDKitRingDecomposerLib.dylib
[ 11%] Built target RingDecomposerLib
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLapsp.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLbitset.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLcycleFams.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLdimacs.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLgraph.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLhandler.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLinfo.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLrelation.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLstack.c.o
[ 11%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLtarjan.c.o
[ 12%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLtesting.c.o
[ 12%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RDLutility.c.o
[ 12%] Building C object External/RingFamilies/CMakeFiles/RingDecomposerLib_static.dir/RingDecomposerLib/src/RingDecomposerLib/RingDecomposerLib.c.o
[ 12%] Linking C static library libRDKitRingDecomposerLib_static.a
[ 12%] Built target RingDecomposerLib_static
[ 12%] Building CXX object External/GA/CMakeFiles/ga.dir/ga/StringChromosome.cpp.o
In file included from /Users/runner/work/rdkit-repro/rdkit-repro/rdkit/External/GA/ga/StringChromosome.cpp:11:
In file included from /Users/runner/work/rdkit-repro/rdkit-repro/rdkit/External/GA/ga/StringChromosome.h:14:
In file included from /Users/runner/work/rdkit-repro/rdkit-repro/rdkit/External/GA/ga/StringChromosomeBase.h:21:
In file included from /Users/runner/work/rdkit-repro/rdkit-repro/rdkit/External/GA/ga/../util/RandomUtil.h:16:
In file included from /Users/runner/work/rdkit-repro/rdkit-repro/rdkit/External/GA/ga/../util/export.h:13:
/Users/runner/work/rdkit-repro/rdkit-repro/rdkit/Code/RDGeneral/RDExportMacros.h:19:10: fatal error: 'boost/config.hpp' file not found
#include <boost/config.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [External/GA/CMakeFiles/ga.dir/ga/StringChromosome.cpp.o] Error 1
make[1]: *** [External/GA/CMakeFiles/ga.dir/all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -457,7 +457,7 @@ endif()
 target_include_directories(rdkit_base INTERFACE
                            $<BUILD_INTERFACE:${RDKit_CodeDir}>
                            $<INSTALL_INTERFACE:${RDKit_HdrDir}>
-                           ${Boost_INCLUDE_DIR}
+                           ${Boost_INCLUDE_DIRS}
                            )
 target_link_libraries(rdkit_base INTERFACE ${BOOST_LIBRARIES})
 

